telegram/ext/_updater.py

```diff
@@ -499,6 +501,11 @@ async def start_webhook(
                 header isn't set or it is set to a wrong token.
 
                 .. versionadded:: 20.0
+            unix (:class:`pathlib.Path` | :obj:`str`, optional): Path to the unix socket file. Path
+                can be empty in which case the file will be created. When using this param, you
+                need to set the :paramref:`webhook_url`!
```

Comment:
please also explain that this is mutually exclusive with listen, port & url_path. The notes on webhook_url & the mutual excluisveness can go into an "Attention" (danger/caution/…) annotation.